tests/bin/docker-test.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env sh
```

Comment:
The `docker-tests` and `tests.sh` pattern was ported rather hastily and lazily.
There's a bunch of lines that were just copied without any thought to it (`see tests.sh` L# 28-34).
Test requirements aren't heavy at all - I recommend just
- removing these wrappers
- requiring users to install the requirements and run pytest locally
- automating the same for CI